File: Beecrowd/1043 Triângulo/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        double a, b, c, per, ar;

        a = sc.nextDouble();
        b = sc.nextDouble();
        c = sc.nextDouble();

        if (((b - c) < a && a < (b + c)) && ((a - c) < b && b < (a + c)) && ((a - b) < c && c < (a + b))) {
            per = a + b + c;
            System.out.printf("Perimetro = %.1f%n", per);
        } else {
            ar = ((a + b) * c) / 2;
            System.out.printf("Area = %.1f%n", ar);
        }
        sc.close();
    }

}

File: Beecrowd/1043 Triângulo/main.py
entrada = [float(i) for i in input().split()]
A = entrada[0]
B = entrada[1]
C = entrada[2]

if ((B - C) < A and A < (B + C)) and ((A - C) < B and  B < (A + C)) and ((A - B) < C and C < (A + B)):
    Per = A + B + C
    print("Perimetro = %.1f" %Per)
else:
    Ar = ((A + B) * C) / 2
    print ("Area = %.1f" %Ar)